test/azure.js

```diff
@@ -0,0 +1,114 @@
+/* global describe, it */
+const assert = require('assert');
+const fs = require('fs');
+const zlib = require('zlib');
+const uploadfs = require('../uploadfs.js')();
+const srcFile = 'test.txt';
+const infile = 'one/two/three/test.txt';
+
+/* helper to automate scraping files from blob svc */
+const _getOutfile = (infile, tmpFileName, done) => {
+  const ogFile = fs.readFileSync(srcFile, {encoding: 'utf8'});
+  uploadfs.copyOut(infile, tmpFileName, {}, (e, res) => {
+    if (e) return console.error("copyOut Error", e);
```

Comment:
This would just hang until test timeout after printing the message, it should be an assert failure.

Reply:
✓